Update suppressed-lists.md (#4490)
SumoLogic/sumologic-documentation: docs/cse/match-lists-suppressed-lists/suppressed-lists.md

When a Record contains a value that matches one or more Suppressed Lists, two fields in the Record get populated:

* `listMatches`. Cloud SIEM adds the names of the Suppressed Lists that the Record matched, and the column values of those lists. For example, if an IP address in a Record matches the SourceIP address in the “vuln_scanners” Suppressed List, the `listMatches` field would look like this: `listMatches: ['vuln_scanners', 'column:SourceIp']`    
* `matchedItems`. Cloud SIEM adds the actual key-value pairs that were matched. For example, continuing the example above, if “vuln_scanners” Match List contained an entry “5.6.7.8”, and the Record’s SourceIp is also “5.6.7.8”, the assuming the SourceIP address in the “vuln_scanners” Suppressed List, the `matchedItems` field would look like this: `matchedItems: [ { value: '5.6.7.8', …other metadata about list item } ]`

Because the information about list matches gets persisted within Records, you can reference it downstream in both rules and search.

> **If any of the entities within the Record match one of the items listed in a suppressed list, then suppressed Signals will be created for that entity across all rules.** This means that these Signals will not contribute to the Activity Score of an entity, nor will the Signals contribute to Insight generation.

Reviewer: ```suggestion
**If any entities in the Record match items from a suppressed list, suppressed Signals will be generated for those entities across all rules**. Consequently, these Signals will not affect the entity's Activity Score or contribute to Insight generation.
```

> For more information about Signal Suppression mechanims, see [About Signal Suppression](/docs/cse/records-signals-entities-insights/about-signal-suppression/).

Reviewer: ```suggestion
For more information about Signal Suppression mechanisms, see [About Signal Suppression](/docs/cse/records-signals-entities-insights/about-signal-suppression/).
```
